wiki/osu!supporter/de.md

```diff
@@ -1,56 +1,157 @@
 
-### Support-Status abfragen
+*Tipp: Kleinere Namensänderungen können beim Account-Support-Team kostenlos beantragt werden — siehe [Hilfecenter/Account § Kann ich den Benutzernamen meines Accounts ändern?](/wiki/Help_centre/Account#name-changes).*
 
-![osu!supporter-Tag mit drei Monaten Restlaufzeit](img/status.jpg?1 "Drei Monate Restlaufzeit des osu!supporter-Tags")
+osu!supporter zu bekommen ist verbunden mit der Fähigkeit, den Benutzernamen beim ersten Mal kostenlos zu ändern, unter den [üblichen Bedingungen](/wiki/Help_centre/Account#name-changes).
 
-*Tipp: Dieser Abschnitt wird nur angezeigt, nachdem du osu!supporter erhalten hast.*
+### osu!direct
 
-Um den Status deines verbleibenden osu!supporter-Tags und die Gesamtsumme der Beiträge und Tag-Käufe zu überprüfen, gehe auf die Seite [Das Spiel unterstützen](https://osu.ppy.sh/home/support) und scrolle nach unten, um einen ähnlichen Bereich wie oben abgebildet zu sehen.
+osu!direct ist das Such- und Downloadmodul für Beatmaps im Spiel, welches nur für Unterstützer verfügbar ist. Es kann über das Hauptmenü erreicht werden, indem man auf das vertikale Label `osu!direct` auf der rechten Seite des Bildschirms klickt. Zusätzlich zu einer eingebauten Beatmap-Auflistung bietet osu!direct andere Wege, an neue Beatmaps zu gelangen, ohne das Spiel zu verlassen:
+
+- Beatmap-Links im Chat, die auf Beatmaps auf der osu!-Webseite zeigen, lösen jetzt im Spiel eine Aufforderung zum Herunterladen aus
+- Automatisches Herunterladen von Beatmaps während des Spielens in einer [Mehrspieler](/wiki/Gameplay/Multiplayer)-Lobby oder beim [Zuschauen](/wiki/Gameplay/Spectating) eines anderen Spielers, als eine [Opt-out-Einstellung](/wiki/Client/Options#integration)
+- Automatische Verlinkung von aktuell gespielten Beatmaps für Zuschauer im Chat-Tab `#spectator`, als eine [Opt-out-Einstellung](/wiki/Client/Options#online)
+
+### Erweiterte Ranglisten
+
+Unterstützer haben Zugriff auf mehrere beatmapspezifische [Ranglisten](/wiki/Beatmap#leaderboards), die sowohl im Spiel als auch auf der Webseite verfügbar sind:
+
+- Globale Ranglisten für jede Kombination an [Spielmodifikationen](/wiki/Game_modifier)
+- Länderspezifische Ranglisten von Spielern, die die gleiche Flagge tragen
+- Freundesranglisten, die die Position eines Spielers auf einer Beatmap verglichen mit seinen Freunden zeigen
+
+### Erhöhte Grenzwerte
+
+osu! bietet eine Lockerung der Grenzwerte für verschiedene Online-Features für Unterstützer an:
+
+| Wert | Normales Limit | osu!supporter-Limit |
+| :-- | :-: | :-: |
+| [Ausstehende Beatmaps](/wiki/Beatmap/Category#work-in-progress-and-pending) | `2 + min(gerankte Beatmaps, 6)`, bis zu **8**[^pending-beatmaps-ref] | `8 + min(gerankte Beatmaps, 12)`, bis zu **20**[^pending-beatmaps-ref] |
+| Online-Beatmap-Favoriten | 100 | 1000 |
+| Anzahl der Freunde | 250 | 500 |
+
+Des Weiteren wird Inhabern von osu!supporter weniger strenge Download-Beschränkungen gewährt.
```

Comment:
`Besitzer` or `Eigner` might be a good alternative too 🤔

Reply:
I think the first sounds better.
To that, shouldn't it be `werden` instead of `wird`, as `Download-Beschränkungen` is plural?
so
`Des Weiteren werden Besitzern von osu!supporter weniger strenge Download-Beschränkungen gewährt.`

Reply:
```suggestion
Des Weiteren werden Besitzern von osu!supporter weniger strenge Download-Beschränkungen gewährt.
```